### ISSUE TITLE
Tutorial Island Giant Rat listed with wrong HP in combat overlay

### ISSUE THREAD

**Issue by Cobodobo** [runelite/runelite#9222]
**Describe the bug**
Tutorial island giant rat shows up with 5 hp in the combat overlay, it actually has 3.

**To Reproduce**
1. Go to tutorial island
2. Enter combat with a giant rat with the combat overlay enabled
3. HP is tracked starting from 5/5

**Expected behavior**
Max HP listed as 5, rat actually has 3.

**Environment (please complete the following information):**
 - OS: Debian
 - RuneLite version: 1.5.27
 - Launcher version: 1.6.2

**Comment by dekvall** [comment]
This bug can be fixed once #8463 is merged by replacing the entry for NPC ID `3313` with
```
  "3313": {
    "name": "Giant rat",
    "hitpoints": 3,
    "combatLevel": 3,
    "slayerLevel": 1,
    "attackLevel": 1,
    "strengthLevel": 1,
    "defenceLevel": 1,
    "rangeLevel": 1,
    "magicLevel": 1,
    "stabDef": -100,
    "slashDef": -100,
    "crushDef": -100,
    "rangeDef": -100,
  }
```
Courtesy of Mod Ash
`What would be the stats you see if you cast monster examine on a rat on tutorial island?`
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Level 1 in the stats other than 3 HP, with -100 defence bonuses in stab, slash, crush and ranged.</p>&mdash; Mod Ash (@JagexAsh) <a href="https://twitter.com/JagexAsh/status/1161260323602599936?ref_src=twsrc%5Etfw">August 13, 2019</a></blockquote>

**Comment by Nightfirecat** [comment]
I've updated [the "Giant rat" wiki page](https://oldschool.runescape.wiki/w/Giant_rat) with the tutorial island variant separated from the other level 3 rats. This issue will be fixed in the client when next our wiki scraper is run automatically, which looks to be in about 1 hour.
